File: Selenium/Project/HRM_Activity2.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class HRM_Activity2 {

    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        String header = driver.findElement(By.tagName("img")).getAttribute("src");
        System.out.println("Header URL: "+header);
        driver.close();
    }


}

File: Selenium/Project/HRM_Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("http://alchemy.hguy.co/orangehrm")
    header = driver.find_element(By.TAG_NAME,"img").get_attribute("src")
    print("Header URL: ",header)
    driver.quit()

